src/layouts/DashboardLayout.vue

```diff
@@ -1,10 +1,14 @@
 <template>
-    <div class="flex h-screen bg-black-alt font-roboto">
-        <Sidebar />
-
-        <div class="flex-1 flex flex-col overflow-hidden">
-            <main class="flex-1 overflow-x-hidden overflow-y-auto">
-                <div class="container mx-auto px-6 py-8">
+    <div class="h-screen flex overflow-hidden bg-gray-50 dark:bg-darkGray-900">
+        <SidebarMobile />
```

Comment:
It should be better to add a single Sidebar component with responsive elements

Reply:
Do you mean to switch rendering & not rendering each component using JS?
Or render single component and only change the style between desktop & moblie?

Reply:
I was thinking of something like this https://www.w3schools.com/howto/howto_css_responsive_header.asp
Plus, Vue.js has conditional rendering that can make the implementation much easier (https://v3.vuejs.org/guide/conditional.html#v-show)